lib/membrane_opus/encoder.ex

```diff
@@ -132,15 +133,34 @@ defmodule Membrane.Opus.Encoder do
   end
 
   @impl true
-  def handle_buffer(:input, %Buffer{payload: data}, _ctx, state) do
-    case encode_buffer(state.queue <> data, state, frame_size_in_bytes(state)) do
-      {:ok, {[], rest}} ->
+  @spec handle_buffer(:input, Membrane.Buffer.t(), any(), %{
+          :input_stream_format => Membrane.RawAudio.t(),
+          :pts => any(),
+          :queue => binary(),
+          optional(any()) => any()
+        }) ::
+          {[{:buffer, {any(), any()}}],
+           %{:pts => any(), :queue => bitstring(), optional(any()) => any()}}
+  def handle_buffer(:input, %Buffer{payload: data, pts: pts}, _ctx, state) do
+    prepared_state =
+      if state.pts == nil do
+        %{state | pts: pts}
+      else
+        state
+      end
```

Comment:
Let's take a look, at what is put in `state.pts` 
1) On the beginning, in `handle_init`, we set `state.pts` to `nil`
2) Then, in the first `handle_buffer`, we set it to `buffer.pts` (let's assume `buffer.pts` is not `nil`)
3) In the calls of `update_state_pts` function we increment `state.pts` by each payload duration
4) Then, in the second call of `handle_buffer` `state.pts` will not be `nil`, so `buffer.pts` will be ignored, am I right? It shouldn't happen, especially if `state.queue` is empty